crates/matrix-sdk/src/event_cache/mod.rs

```diff
@@ -196,6 +196,24 @@ impl EventCache {
         Ok(())
     }
 
+    /// Try to find an event by id in all the rooms.
```

Comment:
Perhaps we should clarify by which ID, if you don't like this form perhaps spelling out the term "event ID" would be better?

```suggestion
    /// Try to find an event by its ID in all the rooms.
```